bin/runtime/src/lib.rs

```diff
@@ -302,11 +280,21 @@ impl OnUnbalanced<NegativeImbalance> for EverythingToTheTreasury {
     }
 }
 
+parameter_types! {
+    // We expect that on average 25% of the normal capacity will be occupied with normal txs.
+    pub TargetSaturationLevel: Perquintill = Perquintill::from_percent(25);
+    // During 20 blocks the fee may not change more than by 100%.
```

Comment:
Nit: 67/1000 implying the fee doubling over 20 blocks sounds totally mysterious. Adding here a bit of explanation (formula? link?) would be a favor to our future selves ;)

Reply:
sure, I referred to the source formulas